plugins/picard-tangoinfo/tangoinfo.py

```
@@ -0,0 +1,71 @@
+"""
+Get genre, performed date and (soon) vocalists and instrumentalists from tango.info
+Uses web scraping since opendata is disabled for now
+"""
+
+PLUGIN_NAME = 'ix5 tango.info'
+PLUGIN_AUTHOR = 'ix5'
+PLUGIN_DESCRIPTION = 'change <em>genre</em> and <em>year</em> fields'
+PLUGIN_VERSION = '0.1'
+PLUGIN_API_VERSIONS = ["0.9.0", "0.12", "1.0.0"]
+
+from picard import log
+from picard.metadata import register_track_metadata_processor
+import re,os
+
+from bs4 import BeautifulSoup
+import urllib2
+
+scraped_data = {}
+
+def scrape_page(barcode):
+    url = ("https://tango.info/0" + str(barcode))
```

It used to fail silently before. I have added a check now.
`if (len(str(metadata["barcode"]))>0):`